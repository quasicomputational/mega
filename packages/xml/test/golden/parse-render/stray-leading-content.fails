  foo bar <x>hello</x>