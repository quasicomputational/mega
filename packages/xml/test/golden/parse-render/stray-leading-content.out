Error: Unexpected text at (1, 3) before the doctype.
